content/articles/whois-privacy.markdown

```diff
@@ -105,14 +97,10 @@ Once you disable the WHOIS Privacy protection, the contact details of your domai
 
     ![Whois privacy in domain list](/files/whoisprivacy-domain-list.png)
 
-1.  Click on the right arrow icon to open the domain feature menu, and click <label>Disable WHOIS Privacy</label>. This takes you to a confirmation page.
+1.  Click on the right arrow icon to open the domain feature menu, to disable the WHOIS Privacy service click on the <label>Disable WHOIS Privacy</label> link.
```

Comment:
The screenshot is a little out of date (there is no arrow icon any more). I think that is fine because it is a tiny change, but let's have the words up to date:

```suggestion
1. Click on the horizontal ellipsis icon to open the domain actions menu.
1. Click on <label>Disable WHOIS Privacy</label>.
```